doc/content/source/kernels/EFieldMagnitudeSource.md

```diff
@@ -1,21 +1,41 @@
 # EFieldMagnitudeSource
 
-!alert construction title=Undocumented Class
-The EFieldMagnitudeSource has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /Kernels/EFieldMagnitudeSource
 
 ## Overview
 
-!! Replace these lines with information regarding the EFieldMagnitudeSource object.
+`EFieldMagnitudeSource` adds a the magnitude of the electric field as a source term. `EFieldMagnitudeSource` assumes the electrostatic approximation for the electric field.
```

Comment:
If this is the magnitude of the electric field then the kernel would need a square root. This is the mangitude squared. 

This also needs to be addressed in the `classDescription` for the kernel

```suggestion
`EFieldMagnitudeSource` adds the magnitude of the electric field squared as a source term. `EFieldMagnitudeSource` assumes the electrostatic approximation for the electric field.
```